packages/compass-shell/package.json

```diff
@@ -58,7 +58,7 @@
     "@mongodb-js/compass-utils": "^0.8.2",
     "@mongodb-js/compass-workspaces": "^0.31.2",
     "@mongosh/browser-repl": "^3.2.0",
```

Comment:
Let's stick to always updating all the `@mongosh` packages if we update one of them, we don't want to end up in a situation where we use mongosh packages that refer to genuinely different mongosh source tree versions

Reply:
yeah that's fair, the other two ended up being bumped in https://github.com/mongodb-js/compass/pull/6747 but good to generally do that as principle